packages/manager/src/features/CloudPulse/Utils/CloudPulseWidgetUtils.test.ts

```diff
@@ -0,0 +1,123 @@
+import { formatPercentage } from 'src/utilities/statMetrics';
+
+import {
+  generateGraphData,
+  generateMaxUnit,
+  getDimensionName,
+  getLabelName,
+  mapResourceIdToName,
+} from './CloudPulseWidgetUtils';
+
+import type { CloudPulseMetricsResponse } from '@linode/api-v4';
+
+it('test generateMaxUnit method', () => {
```

Comment:
Add a test case for some unit which could be rolled up like data is in bytes & maximum possible roll up could be MB, KB etc. Here % percentage we are not supporting in roll up so there will be not any maximum rolled up unit